src/views/Home.vue

```diff
@@ -4,37 +4,71 @@
     p(class="subtitle") BOOKS
```

Comment:
Title use h tag and don't copy class 
```suggestion
    p.subtitle
    | BOOKS
```